src/reference/config/zilla-cli.md

```diff
@@ -24,6 +25,28 @@ The Zilla Runtime command line interface uses the [Zilla Runtime Configuration](
 
 ## Commands
 
+### zilla generate
+
+The `zilla generate` command generates a zilla configuration file from an OpenAPI or AsyncAPI service definition.
+
+The command currently has templates for the following common scenarios:
+- `openapi.http.proxy` - Create a http proxy config based on an OpenAPI service definition.
+- `asyncapi.http.proxy` - Create a http proxy config based on an AsyncAPI service definition.
+- `asyncapi.mqtt.proxy` - Create a mqtt proxy config based on an AsyncAPI service definition.
+
+You have to specify which template to use, the OpenAPI/AsyncAPI service definition as the input file and the output
+file name you want the generated zilla config to be saved.
```

Comment:
It is better to enable text-wrapping in your IDE for markdown files instead of adding newlines in paragraphs. Some .md renderers handle this fine but most static site generators respect the newline in the generated html.

Reply:
fixed